src/__internal__/focus-trap/focus-trap.component.tsx

```diff
@@ -279,7 +279,7 @@ const FocusTrap = ({
     onFocus: updateCurrentFocusedElement,
   });
 
-  // passes focusProps, sets tabIndex and onBlur if no tabIndex has been expicitly set on child
+  // passes focusProps, sets tabIndex and onBlur if no tabIndex has been explicitly set on child
```

Comment:
question: since this isn't related to the commit should we separate it out as it's own chore commit?

Reply:
Yeah, good idea. I'll do that when I come to rebase with master, prior to merging the work.